src/styles/themes.ts

```diff
@@ -10,9 +12,9 @@ export const darkTheme: DefaultTheme = {
   textSecondary: 'var(--color-grey-200)',
   textTertiary: 'var(--color-grey-200)',
 
-  bgPrimary: 'var(--color-black)',
+  bgPrimary: 'var(--color-grey-900)',
```

Comment:
If 2 theme variables end up having the same value, it's probably that we don't use them properly. I'll see how to streamline this situation!